source/index.rst

```diff
@@ -12,34 +12,38 @@ Zppy documentation
 What is zppy?
 =============
 
-Zppy (pronounced zippee) is a post-processing toolchain for E3SM written in 
-Python. The goal of zppy is to speed-up the post-processing of E3SM simulations
+``zppy`` (pronounced "zippee") is a post-processing toolchain for E3SM written in
+Python. The goal of ``zppy`` is to speed-up the post-processing of E3SM simulations
 by automating commonly performed tasks.
 
-Zppy is built around a set of tasks that perform specific actions,
-such as generating (regridded) **time series** or **climarology** files
-using NCO's ncclimo, or generating analysis figures with **E3SM Diags** 
-or **MPAS-Analysis**. Zppy provides a framework to simplify the use of these
+``zppy`` is built around a set of tasks that perform specific actions,
+such as generating (regridded) **time series** or **climatology** files
+using NCO's ``ncclimo``, or generating analysis figures with **E3SM Diags**
+or **MPAS-Analysis**. ``zppy`` provides a framework to simplify the use of these
 tools, especially for long simulations.
 
-The figure below illustrates schematically how zppy functions. 
+..
+   # TODO: redraw figure
+   The figure below illustrates schematically how zppy functions.
 
-Zppy is controlled entirely from a single user provided configuration file 
-("mysimulation.cfg"). This configuration file specifies input and output 
+``zppy`` is controlled entirely from a single user provided configuration file
+(e.g., "mysimulation.cfg"). This configuration file specifies input and output
 directories, as well as the list of tasks to run. Each task operates 
 for a specific model component and over 
-a specified number of years, for example to generate atmosphere climatology 
+a specified number of years, for example generating atmosphere climatology
 files every 20 and 50 years.
 
-Zppy parses the configuration file and generates batch
+``zppy`` parses the configuration file and generates batch
 jobs that are submitted for execution by SLURM. Dependencies between tasks
 are handled and passed to SLURM. Internally, each batch job is created by
 instantiating `Jinja <https://jinja.palletsprojects.com>`_ template scripts 
-usually written in bash, but other languages are supported as well.
+(usually written in bash, but other languages are supported as well).
 
 
-.. image:: figures/zppy_schematic.svg
-   :scale: 80%
+..
+   # TODO: Update schematic to show new dependencies and tasks
+   .. image:: figures/zppy_schematic.svg
```

Comment:
Should we just keep the current schematic? It's technically correct but it doesn't show how E3SM diags can depend on time series and it also doesn't show the global time series. If we just want a simple example diagram, it should be fine. (Image is currently at https://e3sm-project.github.io/zppy/docs/html/index.html).